### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_View Properties_Keyboard: Unable to access the Collapse button in the editor code using Keyboard

### ISSUE THREAD

**Issue by Ashwinipola26** [microsoft/vscode-azurefunctions#2692]
### GitHubTags:
#Azure Tools for VS Code;#A11ySev2;#A11yTCS;#BM-Azure Tools for VS Code-MAC-Feb2021;#DesktopApp;#WCAG2.1.1;#MAC; #Keyboard;#AINotInScope

### OS Details:
macOS Catalina Version 10.15.7

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:45:54.515Z(6 days ago)
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin X64 19.6.0
Azure Function Version: 1.2.1

### Repro Steps:

1. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab till Create a Project and create one project
8. Tab till Created project and press Ctrl+left click and press enter
9. Tab till View Properties
10. Tab till Collapse button which are present in front of code lines in the editor and check whether user is able to access collapse button using keyboard or not.

### Actual Result:
Unable to access the Collapse button in the editor code using Keyboard

### Expected Result:
User should be able to access Collapsed button in the editor code section using keyboard

### User Impact:
Keyboard only users will have difficulty in accessing the Collapse button in the editor code section

### Attachment for Reference:
[Bug_2692_Unable to access the Collapse button in the editor code using Keyboard.zip](https://github.com/microsoft/vscode-azurefunctions/files/6009239/Bug_2692_Unable.to.access.the.Collapse.button.in.the.editor.code.using.Keyboard.zip)

**Comment by ejizba** [comment]
Dupe of https://github.com/microsoft/vscode-azurefunctions/issues/2702
